src/pages/Landing.tsx

```diff
@@ -0,0 +1,13 @@
+import Footer from '../components/molecules/Footer';
+import Header from '../components/molecules/Header';
+
+const Landing = () => {
```

Comment:
We need to pass the children prop to the base page and render it between the header and the footer. Something like

```
({children}) => {
    <header/>
    {children}
    <footer/>
```

Reply:
Done 👍